src/FixedDataTable.js

```diff
@@ -957,12 +957,17 @@ var FixedDataTable = createReactClass({
       children.push(child);
     });
 
-    // Allow room for the scrollbar, less 1px for the last column's border
-    var adjustedWidth = props.width - Scrollbar.SIZE - Scrollbar.OFFSET;
+    var useGroupHeader = children.length > 0 && children[0].type.__TableColumnGroup__ === true;
+    var groupHeaderHeight = useGroupHeader ? props.groupHeaderHeight : 0;
+
+    var tableWidth = props.width;
+    var totalHeightReserved = props.footerHeight + props.headerHeight +
```

Comment:
Have you thought through how total height reserved can be affected by the visibility of the horizontal scrollbar (see line 1130).

That makes figuring this out not fun :(